ibis/expr/datatypes.py

```diff
@@ -32,6 +32,7 @@
 try:
     if sys.version_info >= (3, 6):
         import shapely.geometry
+
```

Comment:
@xmnlab @cpcloud gotta figure out how to better enforce that black is run on every commit, so we don't have these extraneous formatting changes in more recent reviews.

Reply:
I think now black is working properly .. but as it was not working properly in the last months ... some files will be changed now for black. I will open a PR just for black formatting.